events/start_handler.go

```diff
@@ -126,7 +135,6 @@ type ipAddress struct {
 
 type nic struct {
 	IpAddresses []ipAddress `json:"ipAddresses"`
-	Id          string
```

Comment:
Nic's Id is now coming across as an `int`, which was causing another json unmarshal error. Code doesn't need the id, so easiest to just delete it.